server/controllers/inventory/depots/distributions.js

```diff
@@ -15,68 +15,65 @@
 *
 */
 
-var db      = require('../../../lib/db');
-var uuid    = require('node-uuid'),
-  q       = require('q');
+const db = require('../../../lib/db');
+const uuid = require('node-uuid');
+const q = require('q');
 
 // @fixme remove this file
-var journal = {};
+const journal = {};
 
 exports.createDistributions = createDistributions;
 
 /**
 * Create a distribution
 */
 function createDistributions(depotId, body, session) {
-
-  var sql, fmap, queries,
-
-      // create a new document id
-      docId = uuid();
+  // create a new document id
+  const docId = uuid();
 
   // map to create specific distribution
-  fmap = {
-    'service' : createServiceDistribution,
-    'patient' : createPatientDistribution,
-    'loss'    : createLossDistribution,
-    'rummage' : createRummageDistribution
+  const fmap = {
+    service : createServiceDistribution,
+    patient : createPatientDistribution,
+    loss : createLossDistribution,
+    rummage : createRummageDistribution,
   };
 
   // if the type does not map to one of the function above, reject with an error
   // ERR_INVALID_DISTRIBUTION_TYPE
   if (!fmap[body.type]) {
-    return q.reject({
+    q.reject({
```

Comment:
I'm not sure if this code is being used, or it will be rewritten, but this changes the behavior of this function.  If it passes tests though, we can assume that it isn't used.